app-modules/authorization/src/Filament/Pages/Auth/Login.php

```diff
@@ -241,8 +258,10 @@ protected function getForms(): array
                     ->schema([
                         $this->getEmailFormComponent()
                             ->hidden(fn (Login $livewire) => $livewire->needsMFA)
-                            ->dehydratedWhenHidden(),
+                            ->dehydratedWhenHidden()
+                            ->extraInputAttributes(['class' => 'dark:bg-white dark:text-black']),
                         $this->getPasswordFormComponent()
+                            ->extraInputAttributes(['class' => 'dark:bg-white dark:text-black'])
```

Comment:
We actually don't want to do this I think. It was like the second to last item in the ticket that the mockup provided by Product was inaccurate. We don't want the darkmode inputs to be white. We just want it to default to what Filament and the rest of the app does.